### ISSUE TITLE
Version URL is out of date and/or incorrect

### ISSUE THREAD

**Issue by connorads** [lmstudio-ai/lmstudio-bug-tracker#151]
Latest version is `0.3.4`
https://versions-prod.lmstudio.ai/darwin/arm64/0.3.3 is correct
https://versions-prod.lmstudio.ai/darwin/arm64/0.3.4 points to `0.3.3`
In fact everything points to `0.3.3`: https://versions-prod.lmstudio.ai/darwin/arm64/abc123

```json
{"os":"darwin","arch":"arm64","version":"0.3.3","releaseNotes":"## LM Studio 0.3.3 Release Notes\n\n### What's new in 0.3.3\n\n- Configuration Presets\n   - Save your system prompt, inference parameters as a preset.\n   - Each chat can be associated with a preset.\n- Show live token count for `user input` using the selected model's tokenizer\n- Show live token count for `system prompt` using the selected model's tokenizer\n- New buttons at the top of the chat: Clear All Messages, Clone Chat\n- Stats underneath input box: show context fullness % (click on the label to toggle to n_tokens/n_context)\n- QoL: Chat auto-name, skip if time to first token high or tok/sec low\n- Perf: switching between long chats should be faster\n- First model download suggestion: suggest Gemma 2B for machines with less than 12GB of VRAM\n- Copy debug info to clipboard by right clicking the gear icon next to the model\n- Server tab: Code Snippets now have its own top level tab\n- Right click on links shows a context menu with options to open in browser or copy link\n\n#### New settings\n- Open App Settings using `[⌘,]` or `[Ctrl + ,]` from anywhere\n   - Settings icon is now in the bottom right corner of the app, no longer in the sidebar\n   - `[⌘4]` or `[Ctrl + 4]` now opens My Models\n- ThemeSelector (`[⌘K ⌘T]` or `[Ctrl + K, Ctrl + T]` from anywhere)\n- Choose App update channel: `Stable` or `Beta`\n  - Choose between Stable and Beta in the App Settings\n  - Beta versions might have the same version number as Stable versions, but the build number will be higher\n- Settings option to disable double-click to edit a chat message\n\n### Bug Fixes\n\n- Bug fix: [Windows] fix app close/minimize/maximize buttons background overflow\n- Bug fix: Helpful error when the first message is longer than the total context size\n- Bug fix: Button to show / hide intermediate steps in RAG sometimes did not work (fixed)\n- Bug fix: [Developer] Checkbox for showing debug info did not work when unchecked (fixed)\n- Bug fix: Open downloads folder will open the custom downloads folder instead of the default one\n- Bug fix: Updated Jinja template parser to support models using more complex templates\n- Bug fix: Esc wouldn't close some dialogs\n- Bug fix: on first app usage, can't create a chat until a model is loaded\n- Bug fix: downloads popover was covering the settings dialog when both open\n- Bug fix: cmd + R / ctrl + R to regenerate sometimes wouldn't work\n\n#### Migration from 0.2.31 Presets\n- Pre-0.3 presets are automatically migrated to the new format **upon save**. The old files are NOT deleted.\n- Load parameters are not currently included in the new preset format.\n   - Favor editing the model's default config in My Models.\n\n#### Big thank you to community localizers 🙏\n\n- Spanish [@xtianpaiva](https://github.com/xtianpaiva)\n- Norwegian [@Exlo84](https://github.com/Exlo84)\n- German [@marcelMaier](https://github.com/marcelMaier)\n- Turkish [@progesor](https://github.com/progesor)\n- Russian [@shelomitsky](https://github.com/shelomitsky), [@mlatysh](https://github.com/mlatysh), [@Adjacentai](https://github.com/Adjacentai)\n- Korean [@williamjeong2](https://github.com/williamjeong2)\n- Polish [@danieltechdev](https://github.com/danieltechdev)\n- Czech [@ladislavsulc](https://github.com/ladislavsulc)\n- Vietnamese [@trinhvanminh](https://github.com/trinhvanminh)\n- Portuguese (BR) [@Sm1g00l](https://github.com/Sm1g00l)\n- Portuguese (PT) [@catarino](https://github.com/catarino)\n- Chinese (zh-HK), (zh-TW), (zh-CN) [@neotan](https://github.com/neotan)\n- Chinese (zh-Hant) [@kywarai](https://github.com/kywarai)\n- Ukrainian (uk) [@hmelenok](https://github.com/hmelenok)\n- Japanese (ja) [@digitalsp](https://github.com/digitalsp)\n"}
```

This is blocking updating homebrew on https://github.com/Homebrew/homebrew-cask/pull/188102

**Comment by yagil** [comment]
@connorads we haven’t published 0.3.4 as an in-app update yet. Fixing some last minute bugs and doing it today. Will update you here

**Comment by yagil** [comment]
Up now! Sorry for the delay @connorads
